### DISCUSSION TITLE
v1.39.3

### DISCUSSION THREAD

**Discussion by briancao** [umami-software/umami#1606] [General]
## Fixes

This release fixes the following:

- Fixed issue with editing website, #1603 

<hr /><em>This discussion was created from the release <a href='https://github.com/umami-software/umami/releases/tag/v1.39.3'>v1.39.3</a>.</em>

**Comment by xiaoluoboding** [comment]
Vercel + Supabase, I got this error when upgrading to v1.39.3

<img width="1200" alt="CleanShot 2022-10-28 at 17 57 41@2x" src="https://user-images.githubusercontent.com/6118824/198560495-fd0d3fa2-9359-4697-96f1-4f64c5452448.png">
